go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -126,6 +126,21 @@ func (ovn *Controller) configureLoadBalancer(lb, sourceIP string, sourcePort int
 	return nil
 }
 
+// configureLoadBalancerClientIPSessionAffinity updates the load balancer with the specified session
+// timeoutSeconds is a noop until it is implemented in OVN
```

Comment:
Say "`FIXME`", for grep-ability?

Also, should file an issue to remind us that this needs to be fixed. (Or just update #819, and remove the "Fixes #819" from this PR so that stays open.)

Also, if we are expecting the OVN team to implement that, then there should be a bugzilla bug for them if there isn't already, and you can link to that from the ovn-kubernetes issue

Reply:
I need to dig more into this implementation, I remember back in Midonet we can set timeouts to the flows, maybe this is possible to do it right now.
Definitely, this has to be clarified before merge